build.sbt

```diff
@@ -287,8 +284,10 @@ lazy val ctl = project
       CatsFree.value,
       Decline
     ),
-    TaskKey[Unit]("deployTest") := (runMain in Compile).toTask {
-      s" gem.ctl.main --no-ansi --host sbfocstest-lv1.cl.gemini.edu deploy-test ${imageManifest.formatVersion}"
+    TaskKey[Unit]("deployTest") := Def.taskDyn {
+      (runMain in Compile).toTask {
+        s" gem.ctl.main --verbose --host sbfocstest-lv1.cl.gemini.edu deploy-test ${version.value}"
```

Comment:
Does this have any effect if e.g. we are not in VPN and this tries to connect to `sbfoctest-lv1`?

Reply:
I think it is only called automatically for a merge to develop?

Reply:
Yeah it needs to be able to see `sbfocstest-lv1.cl.gemini.edu` and also needs to be able to `ssh` there without a password prompt. Otherwise it will fail and do nothing.

Reply:
But *you* can say `deployTest` (or, more generally, `runMain gem.ctl.main <options>`) at the sbt prompt to deploy to the test machine (or any other machine that has docker set up).